modules/table/src/integrationTest/java/org/apache/ignite/internal/rebalance/ItRebalanceDistributedTest.java

```diff
@@ -766,11 +765,10 @@ private void directUpdateMetastoreRebalanceAssignmentKeys() throws Exception {
 
     private void verifyThatRaftNodesAndReplicasWereStartedOnlyOnce() throws Exception {
         for (int i = 0; i < NODE_COUNT; i++) {
-            verify(
-                    getNode(i).raftManager,
-                    times(1)).startRaftGroupNodeWithoutService(any(), any(), any(), any(), any(RaftGroupOptions.class)
-            );
-            verify(getNode(i).replicaManager, times(1)).startReplica(any(), any(), any(), any(), any());
+            verify(getNode(i).raftManager, timeout(AWAIT_TIMEOUT_MILLIS).times(1))
+                    .startRaftGroupNodeWithoutService(any(), any(), any(), any(), any(RaftGroupOptions.class));
+            verify(getNode(i).replicaManager, timeout(10_000).times(1))
```

Comment:
You can use `AWAIT_TIMEOUT_MILLIS` here as well instead of 10_000

Reply:
Yep, you are right. That's an auto conflict resolution artifact by the way. Anyway fixed.